.github/workflows/release.yml

```diff
@@ -255,9 +263,11 @@ jobs:
 
             ## Note for macOS Users
             The macOS binary is unsigned and it needs to be made runnable using `xattr -c resolc-universal-apple-darwin`.
-          tag_name: ${{ needs.check-version-changed.outputs.PKG_VER }}
-          name: ${{ needs.check-version-changed.outputs.PKG_VER }}
-          draft: true
+          tag_name: ${{ github.ref_name }}
+          name: ${{ github.ref_name }}
+          prerelease: true
+          token: ${{ steps.app-token.outputs.token }}
+          # draft: true
```

Comment:
Is the prerelease needed instead of the draft? If so please delete instead of leaving commented out.